Building quality apps/Naming.md

```diff
@@ -47,7 +47,7 @@ if (checkFileSize(image))
 * Constants should be `ALL_CAPS`.
 * Methods are named after verbs, variables and classes after nouns.
 * XML resource IDs have to use `snake_case`.
-* XML resource IDs should use `camelCase` in a Kotlin project because of [Kotlin synthetic](https://kotlinlang.org/docs/tutorials/android-plugin.html).
+* XML resource IDs should use `camelCase` in a Kotlin project. Historica reason for convention is Kotlin synthetic (deprecated).
```

Comment:
So auto-generated projects use `snake_case` for naming views. The architecture example uses `camelCase` as most of the our projects

Reply:
> auto-generated projects

What do we mean by this? 😄 

Since we're not doing Java-first development anymore, I believe we can leave only the "XML resource IDs should used `camelCase`"

Reply:
When you generate new project it is using `snake_case`, but yea not to complicate we can leave only your suggestion